components/post-editor/PostEditor.tsx

```diff
@@ -25,6 +30,14 @@ interface IProps {
   subTitle?: string;
 }
 
+const cssTextArea =
```

Comment:
this variable with its style is not used anywhere. maybe forgotten?

Reply:
yeah, this was transferred to the custom posttextEditor component